hrm-domain/hrm-core/contact/contactRoutesV0.ts

```diff
@@ -133,23 +133,39 @@ contactsRouter.post(
   publicEndpoint,
   async (req: Request, res: Response, next: NextFunction) => {
     try {
-      const { hrmAccountId, params, can, user, permissions, query } = req;
+      // const { hrmAccountId, params, can, user, permissions, query } = req;
 
-      console.log('params', params); //params will have filters - counsellor, dateFrom, dateTo which will be applied by the ES client
+      // console.log('params', params); //params will have filters - counsellor, dateFrom, dateTo which will be applied by the ES client
       // mocked ES client results with an array of Ids for testing - currently not implemented
       const elasticSearchClient = async () => [26451, 26450, 1];
       const esContactIdsResult = await elasticSearchClient();
 
-      const contactsResponse = await searchContactsByIdCtx(
-        hrmAccountId,
-        esContactIdsResult,
-        query,
-        { can, user, permissions },
-      );
-      res.json(contactsResponse);
+      res.json(esContactIdsResult);
```

Comment:
I think there should be a check with the db first if these ids do exist.

Reply:
Of course, I will implement this appropriately once I have a way to build queries to ES, for now it's just there so you can easily mock the contacts you want back.